bin/Dockerfile.releaser

```diff
@@ -9,15 +9,6 @@ RUN apt-get update && \
                        mercurial \
                        rpm
 
-ENV GORELEASER_VERSION=0.116.0 \
-    GORELEASER_SHA=34b7e3b843158bd0714d1be996951685496491adab4524fb1198ae144ab06ba3 \
-    GORELEASER_DOWNLOAD_FILE=goreleaser_Linux_x86_64.tar.gz
-
-ENV GORELEASER_DOWNLOAD_URL=https://github.com/goreleaser/goreleaser/releases/download/v${GORELEASER_VERSION}/${GORELEASER_DOWNLOAD_FILE}
-
-RUN wget "${GORELEASER_DOWNLOAD_URL}" && \
-    echo "$GORELEASER_SHA $GORELEASER_DOWNLOAD_FILE" | sha256sum -c - && \
-    tar -xzf "$GORELEASER_DOWNLOAD_FILE" -C /usr/bin/ goreleaser && \
-    rm "$GORELEASER_DOWNLOAD_FILE"
+RUN curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
 
```

Comment:
I just commented on this dockerfile in your cyberark/sidecar-injector#36 PR - and it occurs that I was commenting below on the _tag_ `cyberark/goreleaser` and you may not remember that we had an actual project to build a custom goreleaser image: https://github.com/conjurinc/goreleaser-docker/blob/master/Dockerfile

now we have this in two projects - we should definitely try to contribute the fix upstream sooner rather than later, and file tech debt issues in these two repos to update the image used. if we can't fix upstream, I suppose we'll have to resurrect our custom image :/

Reply:
I get 404 on https://github.com/conjurinc/goreleaser-docker/blob/master/Dockerfile. We should definitely have a custom image to avoid the duplication (both repos are building the same image). Contributing to goreleaser would be nice bonus.

Reply:
ah, it's private archived and I can't change the access rights. it does exist!!

I was hoping to move away from bespoke goreleaser images - I thought we had made good progress, then these PRs brought to light for me that there were still issues. sounds like you're taking a good approach moving forward, though :)